source/shared/core_sqlsrv.h

```diff
@@ -1310,7 +1343,11 @@ const int SQLSRV_DEFAULT_SIZE = -1;     // size given for an output parameter th
 const unsigned int QUERY_TIMEOUT_INVALID = 0xffffffff;
 
 // special buffered query constant
+#ifndef _WIN32
```

Comment:
I agree with @david-puglielli 
Perhaps we should find a constant that works for both?